### ISSUE TITLE
Docker-compose file

### ISSUE THREAD

**Issue by lambarena** [beeworking/voyant#10]
I am getting the following error when I run docker-compose:

`envsubst: error while reading "standard input": Is a directory`

Which points to this line in docker-compose.yml:
`command: /bin/bash -c "envsubst < /etc/nginx/conf.d/nginx.template > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'"`

I tried removing the envsubst line completely and manually modify the nginx conf to 0.0.0.0:8000 but that didn't work.

Here's the docker-compose.yml:

```
web:
  image: nginx:latest
  ports:
    - "8080:8080"
  volumes:
    - nginx.template:/etc/nginx/conf.d/nginx.template
    - ./clients/web/:/html/
  links:
    - api
  command: /bin/bash -c "envsubst < /etc/nginx/conf.d/nginx.template > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'"

api:
  build: .
  volumes:
    - .:/app/
  links:
    - db

db:
  image: mongo:3.2
  volumes:
   - ./volume/db:/data/db
```

Reference:
[Issue #496](https://github.com/docker-library/docs/issues/496)

**Comment by volnt** [comment]
Hello,

Thank you for opening this issue.

I am not able to reproduce your problem, have you tried updating your nginx image (`docker pull nginx`) ?

**Comment by lambarena** [comment]
Hi volnt,

I just tried pulling the image and it looks like I already have the latest version:

```
Using default tag: latest
latest: Pulling from library/nginx
Digest: sha256:0fe6413f3e30fcc5920bc8fa769280975b10b1c26721de956e1428b9e2f29d04
Status: Image is up to date for nginx:latest
```

I'll try on a different server. I am using Digital Ocean's docker image.

Regards

**Comment by volnt** [comment]
@lambarena Hello,

It's 5 months later but using the same docker-compose.yml file as this project on another one I got the same issue that you mentionned. I had totally forgot about this issue though but I found it while looking for answers on Google !

Anyway, I found why it does that and how to fix it. On the latest version of docker-compose relative paths should start with a `./` otherwise it will fail to find the file on the host and create an empty directory on the container.

I updated the docker-compose.yml accordingly (https://github.com/beeworking/voyant/commit/510c52415ad1459eae9b61e3c60789a3fc630d96).

Reference : https://github.com/docker/compose/issues/424

Thank you again for your Issue,

Florent
